source/slang/glsl.meta.slang

```diff
@@ -2825,21 +2825,1988 @@ public uint rayQueryGetIntersectionTypeEXT(rayQueryEXT q, bool committed)
     return 0;
 }
 
+// TODO: implementation of built-in variables; proper tests; these are stubs
+// likley related to the following issue since GLSL adds new 
+// 'system' variables: https://github.com/shader-slang/slang/issues/411
+
+#define glsl_caps_shader_subgroup(param1) __glsl_extension(param1) [require(glsl)] 
+#define spirv_caps_shader_subgroup(param1) __spirv_version(param1) [require(spirv)] 
+
+//macros used to make implementation and modification reasonable under regex
+#define macroAnyVecGenericBuiltIn __generic<T : __BuiltinType, let N : int>
+#define macroAnyVecGenericLogical __generic<T : __BuiltinLogicalType, let N : int>
+#define macroAnyVecGenericArithmetic __generic<T : __BuiltinArithmeticType, let N : int>
+#define macroAnyVec vector<T, N>
```

Comment:
Let's not use define macro as much as we can.
It may seem verbose but we don't want to scroll up and down to see what the macro define might be doing something wrong.